.ksql/BUILDING_SNAPSHOT_OSLO_STREAM_FLATTEN_GEOLOCATION.ksql

```diff
@@ -0,0 +1,15 @@
+CREATE OR REPLACE STREAM BUILDING_SNAPSHOT_OSLO_STREAM_FLATTEN_GEOLOCATION
+WITH (KAFKA_TOPIC='stg.building.snapshot.oslo.flatten.geolocation', PARTITIONS=6, VALUE_FORMAT='JSON_SR') AS 
+SELECT
+  REDUCE(SPLIT(URL_EXTRACT_PATH(MESSAGEKEY), '/'), '', (s,x) => x) as MSGKEY,
+  IDENTIFICATOR->ID IDENTIFICATOR_ID,
+  IDENTIFICATOR->NAAMRUIMTE IDENTIFICATOR_NAAMRUIMTE,
+  CAST(IDENTIFICATOR->OBJECTID AS INTEGER) IDENTIFICATOR_OBJECTID,
```

Comment:
Moet deze cast hier ook weg? Zie PR Luk

Reply:
depends on whether the cast is correct or not